src/components/DataDictionary.tsx

```
@@ -222,6 +375,7 @@ const LayoutSidebar = styled.aside`
   z-index: 0;
 `;
 
+// @ts-ignore
```

why do we have to ignore these now?

---

Good catch. They error for me even if I drop my typescript version: 

https://looker.slack.com/archives/C9NHFLY0G/p1596218068427800

Wanted to ask if ignoring these will suppress errors inside